daemon/api_test.go

```diff
@@ -1336,22 +1336,34 @@ func (s *apiSuite) TestConnectPlugSuccess(c *check.C) {
 		"status_code": 200.0,
 		"type":        "sync",
 	})
-	for slot, plugs := range d.interfaces.ConnectedSlots("consumer") {
-		c.Check(slot.Snap, check.Equals, "consumer")
-		c.Check(slot.Name, check.Equals, "slot")
-		for _, plug := range plugs {
-			c.Check(plug.Snap, check.Equals, "producer")
-			c.Check(plug.Name, check.Equals, "plug")
-		}
-	}
-	for plug, slots := range d.interfaces.ConnectedPlugs("producer") {
-		c.Check(plug.Snap, check.Equals, "producer")
-		c.Check(plug.Name, check.Equals, "plug")
-		for _, slot := range slots {
-			c.Check(slot.Snap, check.Equals, "consumer")
-			c.Check(slot.Name, check.Equals, "slot")
-		}
-	}
+	c.Assert(d.interfaces.Interfaces(), check.DeepEquals, &interfaces.Interfaces{
+		Plugs: []*interfaces.Plug{
+			&interfaces.Plug{
+				Snap:      "producer",
+				Name:      "plug",
+				Interface: "interface",
+				Connections: []interfaces.SlotRef{
+					{
+						Snap: "consumer",
+						Name: "slot",
+					},
+				},
```

Comment:
This is wasting a lot of screen state. The 6 lines above are actually:

        Connections: []interfaces.SlotRef{{Snap: "consumer", Name: "slot"}}

!